Hadrons/Modules/MNPR/FourQuarkFullyConnected.hpp

```diff
@@ -126,23 +126,23 @@ template <typename FImpl1, typename FImpl2>
 void TFourQuarkFullyConnected<FImpl1, FImpl2>::execute()
 {
     LOG(Message) << "Computing contractions '" << getName()
-        << "' using source propagators '" << par().Sin << "' and '" << par().Sout << "'"
+        << "' using source propagators '" << par().qIn << "' and '" << par().qOut << "'"
         << std::endl;
 
-    PropagatorField1 &Sin = envGet(PropagatorField1, par().Sin);
-    PropagatorField2 &Sout = envGet(PropagatorField2, par().Sout);
+    PropagatorField1 &qIn = envGet(PropagatorField1, par().qIn);
+    PropagatorField2 &qOut = envGet(PropagatorField2, par().qOut);
 
     envGetTmp(LatticeSpinColourMatrix, bilinear);
```

Comment:
This needs a FImpl-derived type as well but it is a similar case as the SpinColourSpinColourMatrix. I can write the data type for that.